README.md

```diff
@@ -45,4 +45,23 @@ allow {
     input.method == "GET"
     input.user == "bob"
 }
-```
\ No newline at end of file
+```
+
+## Trying the example
+- As a manager, create a car (this should be allowed):
+```
+curl -H 'Authorization: alice' -H 'Content-Type: application/json' \
+    -X PUT localhost:8080/cars/test-car \
+    -d '{"model": "Toyota", "vehicle_id": "357192", "owner_id": "4821", "id": "test-car"}'
+```
+
+- As a car admin, try to delete a car (this should be denied):
```

Comment:
Hmm, a "car admin" is a role I think is a little difficult to relate to, and why it wouldn't be allowed to DELETE cars. Could we come up with a better role name?

Reply:
I took it from there https://github.com/open-policy-agent/example-api-authz-python

Reply:
OK. I'd probably prefer that to be changed too, to be honest. But let's keep the policy as it is for now then.